Tiltfile

```diff
@@ -187,6 +187,16 @@ def build_node_yaml():
                     "0xde0036a9600559e295d5f6802ef6f3f802f510366e0c23912b0655d972166017::state::WormholeMessageHandle",
                 ]
 
+            if sui:
+                container["command"] += [
+                    "--suiRPC",
+                    "http://sui:9002",
+                    "--suiAccount",
```

Comment:
can you put a note here saying that the mainnet flags will need the additional argument? whatever's here is going to be used as a template for the mainnet instructions and it's better to catch that early

Reply:
How about this?

Reply:
perfect